media/pages/sk/nahlasit-chybajucu-instituciu/page.html

```diff
@@ -0,0 +1,3 @@
+{# vim: set filetype=htmldjango shiftwidth=2 :#}
+
+<iframe src="https://forms.gle/pQKEHgAqvNZfF5ue9" width="100%" height="840" frameborder="0"></iframe>
```

Comment:
@martinmacko47 V navode bol `src` link vygenerovany v google drive vo formate dlhej url adresy a mal tam tiez este pridany attrtbiut `?embed=true". Nevsimol som si rozdiel vo funkcionalite s alebo bez `embed`. Nevies naco to sluzi? Vyplnil som dvakrat formular, skoncilo to spravou "Your response has been send.", alebo nieco podobne, takze by sa to malo zapracovat. Nemam vsak pristup k formulari, mohol by si mi spristupnit formular.

Reply:
@viliambalaz Vyzera, ze funguje, tak asi `?embed=true` netreba. Ak prestane fungovat, pridame potom.